applications/java/src/build.gradle

```diff
@@ -1,2 +1,7 @@
 apply plugin: 'java'
 apply plugin: 'war'
+
+war {
+    baseName = 'java-app'
```

Comment:
you could have kept it as `app` to avoid changes in the workshop. I do want to show case the rewrite path functionality with some app, it may be handy for a lot of enterprises.

Reply:
to make it same context root across the workshop